### DISCUSSION TITLE
Is there a simple way to detect abnormal behavior?

### DISCUSSION THREAD

**Discussion by muhammadrizki985** [blakeblackshear/frigate#8349] [General]
I have an automation where an indicator light turns on when someone is in front of my house.

The issue is that the indicator light often turns on and off frequently due to the high number of people passing by my house.

I want to create a system that can distinguish between two things:

1) people who are just passing by (normal behavior)
2) people who intend to visit or there's a package delivery (abnormal behavior)

**Comment by wizu** [comment]
You should be able to do it with zones
https://docs.frigate.video/configuration/zones
https://docs.frigate.video/guides/stationary_objects
